src/lib/components/Editor/GeneralEditor.tsx

```diff
@@ -30,6 +31,9 @@ const GeneralEditor = React.forwardRef<ReactQuill, IQuillEditorProps>(
               setText(content);
               dispatchContent(content);
             }}
+            onKeyPress={() => {
+              dispatchDone();
+            }}
```

Comment:
btw
```suggestion
            onKeyPress={dispatchDone}
```
should be enogh, pls check